test/e2e/deployment.go

```diff
@@ -34,13 +34,13 @@ var _ = Describe("Deployment", func() {
 	It("deployment should create new pods", func() {
 		testNewDeployment(f)
 	})
-	It("RollingUpdateDeployment should delete old pods and create new ones [Flaky]", func() {
+	It("deployment should delete old pods and create new ones with RollingUpdateDeployment strategy [Flaky]", func() {
```

Comment:
We have a lot of PRs in flight. Please avoid unrelated changes like this, since they'll likely cause other PRs to need rebases.